node/task.ts

```diff
@@ -1157,6 +1159,27 @@ export function globFirst(pattern: string): string {
     return matches[0];
 }
 
+export function findFiles(includePattern: string[], excludePattern: string[], workingDir?: string): string[] {
```

Comment:
@ericsciple ping. We should close on this asap, as we have too much dependency on this functionality across different tasks :)